datafeed/validations.go

```diff
@@ -0,0 +1,98 @@
+package datafeed
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/datafeed/proto"
+	"github.com/0xPolygon/polygon-edge/helper/hex"
+)
+
+type verifyApiResponse struct {
+	Outcome   int32
+	Timestamp int64
+}
+
+func (d *DataFeed) verifyMarketOutcome(payload *proto.DataFeedReport, verifyApiHost string) (bool, error) {
+	requestURL := fmt.Sprintf("%s/api/outcome/%s", verifyApiHost, payload.MarketHash)
```

Comment:
I think we should have `/api/outcome/` as part of the VerifyAPIURI string passed in from the config rather than appending it here.

Then we can just append the payload.MarketHash to the specified URI for now.